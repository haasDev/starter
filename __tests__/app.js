import React from 'react';
import App from '../src/app';
import snapshot from './snapshot';

describe('App', () => {
  it('should render', () => {
    snapshot(<App />);
  });
});
