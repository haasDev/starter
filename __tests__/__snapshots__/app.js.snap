exports[`App should render 1`] = `
<div
  className="app">
  <div
    className="error">
    
  </div>
  <div
    className="data-entry">
    <div>
      <div
        className="data-input">
        <label>
          Project Name
        </label>
        <input
          autoFocus={undefined}
          className="form-control name-input"
          name="name"
          onChange={[Function]}
          onKeyPress={undefined}
          type="text"
          value="" />
      </div>
      <div
        className="data-input">
        <label>
          Project Description
        </label>
        <input
          autoFocus={undefined}
          className="form-control description-input"
          name="description"
          onChange={[Function]}
          onKeyPress={[Function]}
          type="text"
          value="" />
      </div>
      <button
        className="add-btn btn"
        onClick={[Function]}>
        Add
      </button>
    </div>
    <a
      href="#display">
      Show Projects
    </a>
  </div>
</div>
`;
